### DISCUSSION TITLE
Clarification requested bug or user error

### DISCUSSION THREAD

**Discussion by KyleC69** [OmniSharp/omnisharp-vscode#7389] [Q&A]
Greetings,  I have a situation that started a couple of updates ago that I have been unable to solve. I am getting floods of what look like informational log entries to my journal from Language Server and Code Server. They are not error messages, I will attach a screenshot. 
   I am using VS Code in Portable mode and I have cleared the data/extensions and app folders one at a time and it did not change anything. I have sifted through the settings, and think all logging is off. I have created new profiles, tried sync on and off. Same results with new bare proj. I don't want to report a bug if it's my error, If anyone can offer a suggestion I'd appreciate it.  -- Kyle
![Screenshot from 2024-07-24 05-50-05](https://github.com/user-attachments/assets/4c869d5c-6304-461b-8fd4-956189bd93b5)

vscode 1.91 (portable)
dotnet 7,8,9 installed proj set to 8
Ubuntu 24.04
Unity 7.7

**Comment by dibarbet** [comment]
@KyleC69 always feel free to report a bug, we'll figure out if it is or is not :).  We don't always see discussions as quickly as bugs.

For these logs, they appear to be coming from C# Dev Kit.  I'm not familiar with where these show up, you would likely need to file something here -https://github.com/microsoft/vscode-dotnettools/issues
